### PR TITLE
[#3141] Fix layout issues concerning resource specific metadata cutoff

### PR DESCRIPTION
**Pull request by Maurier**
#3141 Contains layout fixes for resource specific content of deprecated resource types.

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: hs_app_netCDF/forms.py [hunk 1]
```diff
@@ -234,6 +234,7 @@ class VariableValidationForm(forms.Form):
         """
         {% load crispy_forms_tags %}
             {% if variable_formset.forms %}
+            <div class="col-xs-12">
```
Comment (sblack-usu) [inline hs_app_netCDF/forms.py]: Indent this block.

File: hs_app_netCDF/models.py [hunk 2]
```diff
@@ -571,9 +571,7 @@ def get_update_netcdf_file_html_form(self):
         form_action = "/hsapi/_internal/netcdf_update/{}/".\
             format(self.resource.short_id)
         style = "display:none;"
-        if self.is_dirty:
```
Comment (sblack-usu) [inline hs_app_netCDF/models.py]: Could you explain to me why this check is going away?

Comment (Maurier) [reply hs_app_netCDF/models.py]: It was used to populate a context variable that is no longer needed and was removed in this branch.

Comment (Maurier) [reply hs_app_netCDF/models.py]: Correction. It was used to render inline CSS which was replaced by a utility class.

File: hs_core/page_processors.py [hunk 3]
```diff
@@ -208,7 +208,6 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                    'discoverable': discoverable,
                    'resource_is_mine': resource_is_mine,
                    'allow_resource_copy': allow_copy,
-                   'is_resource_specific_tab_active': False,
```
Comment (sblack-usu) [inline hs_core/page_processors.py]: Is this just cleanup you saw while in the code?  Or is this relevant to the metadata?

Comment (Maurier) [reply hs_core/page_processors.py]: This was part of the Resource Specific metadata content. It is code cleanup of the actual HTML I was modifying.

File: hs_core/templates/pages/genericresource.html [hunk 4]
```diff
@@ -779,8 +779,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                     {% if cm.resource_type != "CompositeResource" and cm|resource_type|lower != "collection resource" %}
                         <div class="col-xs-12 content-block">
                             <h3>Resource Specific</h3>
-                            <div class="{% if not is_resource_specific_tab_active %}tab-pane{% else %}tab-pane active{% endif %}"
```
Comment (Maurier) [inline hs_core/templates/pages/genericresource.html]: @sblack-usu This is where the context variable was used and it's no longer needed. This is the container of resource specific metadata.

File: hs_app_timeseries/templates/pages/timeseriesresource.html [hunk 5]
```diff
@@ -25,29 +25,28 @@
...
-                                {% endif %}
-                                <tr><th class="text-muted">Elevation M </th><td>{{ site.elevation_m }}</td></tr>
-                                <tr><th class="text-muted">Elevation Datum </th><td>{{ site.elevation_datum }}</td></tr>
-                                <tr><th class="text-muted">Site Type </th><td>{{ site.site_type }}</td></tr>
-                                <tr><th class="text-muted">Latitude </th><td>{{ site.latitude }}</td></tr>
-                                <tr><th class="text-muted">Longitude </th><td>{{ site.longitude }}</td></tr>
-                            </tbody>
-                        </table>
-                        <hr style="border:0">
+                            <table class="custom-table space-bottom">
```
Comment (engineSound) [inline hs_app_timeseries/templates/pages/timeseriesresource.html]: Actual change is adding "space-bottom" class style.

File: ref_ts/templates/pages/reftimeseriesresource.html [hunk 6]
```diff
@@ -1,114 +1,107 @@
...
                 </div>
-                <div>
-                    {% if quality_level and quality_level.code and quality_level and quality_level.definition %}
-                        <h4><strong>Quality Control Level:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        {% if quality_level.code %}
+            {% endif %}
+
+            {% if quality_level and quality_level.code or quality_level and quality_level.definition %}
+                <legend>Quality Control Level</legend>
```
Comment (engineSound) [inline ref_ts/templates/pages/reftimeseriesresource.html]: Good choice using <legend> for a subtitle.

Comment (engineSound) [reply ref_ts/templates/pages/reftimeseriesresource.html]: The subtitle "Extent". Should it also be changed to <legend> for consistency purpose?

File: ref_ts/templates/pages/reftimeseriesresource.html [hunk 7]
```diff
@@ -1,114 +1,107 @@
...
+                    {% endif %}
+                    {% if site.name %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Name:</strong></div>
                             <div class="col-sm-8">{{ site.name }}</div>
                         </div>
-                        {%  endif %}
-                        {% if site.net_work %}
+                    {% endif %}
+                    {% if site.net_work %}
```
Comment (engineSound) [inline ref_ts/templates/pages/reftimeseriesresource.html]: I noticed you made indentation change here.

File: hs_app_netCDF/templates/pages/netcdfresource.html [hunk 8]
```diff
@@ -4,110 +4,102 @@
...
+                            <tr><th class="text-muted">Coordinate String </th><td style="word-break:break-all" >{{ original_coverage.projection_string_text}}</td></tr>
+                        {% endif %}
+                    </tbody>
+                </table>
+                {% endif %}
 
-                            {% if original_coverage.eastlimit %}
-                                <tr><th class="text-muted">East </th><td>{{ original_coverage.eastlimit }}</td></tr>
-                            {% endif %}
+                <h4>Extent</h4>
```
Comment (engineSound) [inline hs_app_netCDF/templates/pages/netcdfresource.html]: Why not use <legend> here?

File: ref_ts/templates/pages/reftimeseriesresource.html (hunk 7 above)
Comment (Maurier) [reply ref_ts/templates/pages/reftimeseriesresource.html]: Correct. The indentation we had before was off.

**Comment by engineSound** [comment]
@Maurier Impressive fast turnaround.

File: ref_ts/templates/pages/reftimeseriesresource.html (hunk 6 above)
Comment (Maurier) [reply ref_ts/templates/pages/reftimeseriesresource.html]: We could, I'll try to make this change after the release. For now we'll leave it like this since the only difference is a bit of spacing.

File: hs_app_netCDF/templates/pages/netcdfresource.html (hunk 8 above)
Comment (Maurier) [reply hs_app_netCDF/templates/pages/netcdfresource.html]: We could, I'll try to make this change after the release. For now we'll leave it like this since the only difference is a bit of spacing.

**Comment by Lizabrazil** [comment]
@Maurier can you clarify is this closed/good to go?

**Comment by Maurier** [comment]
@Lizabrazil -This was included with release 1.19. This is the same code which will be included in the develop branch as well.

**Comment by sblack-usu** [comment]
@Maurier - This work was merged in already when master was merged into develop.  Not a big deal that you merged this PR, but it was already merged.

**Comment by Maurier** [comment]
@sblack-usu That's weird, the develop branch still showed as outdated to me. It shows differences in the Files Changed tab of this PR.

**Comment by cuahsimarko** [comment]
There's some weird stuff going on. Let's be cautious going forward and consider `diff origin/develop` when playing with feature branches for any of the open 25 PRs (as of Feb 25 4:27 PM EST)

**Comment by sblack-usu** [comment]
@cuahsimarko let's not jump to conclusions scare anyone about this.  I still haven't fetched develop since the merge and see these changes from the master merge.  It could just be something to do with github's diff viewer not refreshing the view.  I don't see anything wrong with develop branch.

**Comment by cuahsimarko** [comment]
@sblack-usu I think develop is fine. I was just trying to save downstream trouble my making sure the feature branches don't introduce anything odd to develop. I think it's all inline, it's just hard to test things locally if they are very out of date with develop, based on some of the core changes.
